Makefile

```diff
@@ -12,18 +12,27 @@ IDENTIFIER=$(GOOS)-$(GOARCH)
 help:  ## Show available options with this Makefile
 	@grep -F -h "##" $(MAKEFILE_LIST) | grep -v grep | awk 'BEGIN { FS = ":.*?##" }; { printf "%-15s  %s\n", $$1,$$2 }'
 
+web:
+	@cd dashboard/web && yarn build && cp -a ./dist ../../web
+
+yarn:
+	@rm -rf web
+	@cd dashboard/web && yarn build && cp -a ./dist ../../web
+
 .PHONY: build
-build:
+build: 
```

Comment:
We should also add `web` as build dependencies because Docker will use `make build` directly